oasislmf/cmd/api.py

```diff
@@ -42,14 +42,32 @@ def load_credentials(login_arg, logger=None):
 
     try:
         api_login = {}
-        api_login['username'] = six.input('Username: ')
+        api_login['username'] = six.moves.input('Username: ')
```

Comment:
It is probably better style to have an import like
   
    ...
    from six.moves import input
    ...

and then have

    ...
    api_login['username'] = input('Username: ')
    ...

But it is fine from a functional POV.